pkg/apis/marketplace/v1alpha1/operatorsource_types.go

```diff
@@ -13,19 +13,6 @@ const (
 	OpSrcFinalizer = "finalizer.operatorsources.marketplace.redhat.com"
 )
 
-// +genclient
-// +genclient:noStatus
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
-// OperatorSource is the Schema for the operatorsources API
-// +k8s:openapi-gen=true
-type OperatorSource struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              OperatorSourceSpec   `json:"spec,omitempty"`
-	Status            OperatorSourceStatus `json:"status,omitempty"`
-}
-
 // Only type definitions go into this file.
```

Comment:
Did you have to move this manually to get it to work?

Reply:
Not as far as I know. I didn't rerun the sdk to generate these, just copied them from your working branch. Would you prefer that we rerun the sdk again to generate this?